receiver/sqlserverreceiver/README.md

```diff
@@ -36,6 +38,11 @@ Windows-specific options:
 - `computer_name` (optional): The computer name identifies the SQL Server name or IP address of the computer being monitored.
   If specified, `instance_name` is also required to be defined. This option is ignored in non-Windows environments.
 
+Query-level metric specific options (only useful when query-level metrics/logs collection are enabled):
+- `lookback_time` (optional, example = `60`, default = `2 * collection_interval`): The time window (in second) in which to query for metrics.
```

Comment:
Why the `lookback_time` is two times of the `collection_interval`? Would this make us receive duplicate data?

Reply:
Maybe we can include a comment like this to explain why we need `lookback_time`

> Queries that finished executing outside of the lookback window are not included in metrics collection. Increase the lookback window (in seconds) to capture queries that run less frequently.